src/SpectatorView.Unity/Assets/SpectatorView.Editor/Scripts/CompositorWindowBase.cs

```diff
@@ -75,7 +76,7 @@ protected void HolographicCameraNetworkConnectionGUI(string deviceTypeLabel, Dev
                     deviceInfo.NetworkManager != null &&
                     deviceInfo.NetworkManager.IsConnected &&
                     spatialCoordinateSystemParticipant != null &&
-                    spatialCoordinateSystemParticipant.PeerDeviceHasTracking &&
+                    spatialCoordinateSystemParticipant.PeerDeviceTrackingState != TrackingState.LostTracking &&

```

Comment:
>PeerDeviceTrackingState != TrackingState.LostTracking [](start = 55, length = 53)

I think this should perhaps be "PeerDeviceTrackingState == TrackingState.Tracking"